### PR TITLE
Add list of features to README

### PR DESCRIPTION
**Pull request by jeddy3**
A top-level overview so potential new users can see what's available without having to pick through the rest of the docs.

I think the quantity of rules, latest css syntax support, being unopinionated, shareable configs, plugin and the options validator are the main features. Have a missed any, or would you replace any?

@davidtheclark I'd appreciate your language skills in fine-tuning this one :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 > Modern CSS linter.
 
-Supports the latest CSS syntax and features, including custom properties, range context for media features, calc() and nesting.
+## Features
+
+* _Over 85 rules_ - from stylistic rules, such as checking the spacing around the colon in every declaration, to rules that catch subtle coding mistakes, such as invalid hex colors or [overriding shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases).
```
Comment (MoOx) [inline README.md]: I think this number is not a good idea and I would go for a approximation like "almost a hundred of rules" (to avoid having to update it.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+* _Support for the latest CSS syntax and features_ - including custom properties, range context for media features, calc() and nesting.
```
Comment (MoOx) [inline README.md]: "CSS syntax" seems enough. I think we do not support "features".

**Comment by MoOx** [comment]
Nice changes!

File: README.md (hunk 1 above)
Comment (jeddy3) [inline README.md]: Agreed :)

File: README.md (hunk 2 above)
Comment (jeddy3) [inline README.md]: Yeap.

**Comment by jeddy3** [comment]
I'll wait on @davidtheclark  feedback before merging this.

File: README.md [hunk 3]
```diff
@@ -2,7 +2,14 @@
...
 
-Supports the latest CSS syntax and features, including custom properties, range context for media features, calc() and nesting.
+## Features
+
+* _Nearly a hundred rules_ - from stylistic rules, such as checking the spacing around the colon in declarations, to rules that catch subtle coding mistakes, such as invalid hex colors or [overriding shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases).
+* _Support for the latest CSS syntax_ - including custom properties, range context for media features, calc() and nesting.
+* _Completely unopinionated_ - only enable the rules you want and configure them with options to tailor the linter to your needs.
+* _Sharable configs_ - if you don't want to craft your own config you can use a shareable config, for example one that checks against the [SuitCSS code style](https://github.com/suitcss/suit/blob/master/doc/STYLE.md).
+* _Support for plugins_ - it's easy to create your own rules and add them to the linter.
+* _Options validator_ - so you can be confident that config is valid and linter is working correctly.
```
Comment (davidtheclark) [inline README.md]: I'd shorten this to "so you can be confident that your config is valid."

File: README.md [hunk 4]
```diff
@@ -2,7 +2,14 @@
 
 > Modern CSS linter.
 
-Supports the latest CSS syntax and features, including custom properties, range context for media features, calc() and nesting.
+## Features
+
+* _Nearly a hundred rules_ - from stylistic rules, such as checking the spacing around the colon in declarations, to rules that catch subtle coding mistakes, such as invalid hex colors or [overriding shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases).
+* _Support for the latest CSS syntax_ - including custom properties, range context for media features, calc() and nesting.
+* _Completely unopinionated_ - only enable the rules you want and configure them with options to tailor the linter to your needs.
+* _Sharable configs_ - if you don't want to craft your own config you can use a shareable config, for example one that checks against the [SuitCSS code style](https://github.com/suitcss/suit/blob/master/doc/STYLE.md).
```
Comment (davidtheclark) [inline README.md]: I think we could cut this down to: "if you don't want to craft your own config you can use a shareable config."

**Comment by davidtheclark** [comment]
Great change. I made a couple of comments. Feel free to merge when you're happy with it.
